.ci/workchains.py

```diff
@@ -68,8 +68,8 @@ def a_magic_unicorn_appeared(self, node):
     @process_handler(priority=400, exit_codes=ArithmeticAddCalculation.exit_codes.ERROR_NEGATIVE_NUMBER)
     def error_negative_sum(self, node):
         """What even is a negative number, how can I have minus three melons?!."""
-        self.ctx.inputs.x = Int(abs(node.inputs.x.value))
-        self.ctx.inputs.y = Int(abs(node.inputs.y.value))
+        self.ctx.inputs.x = Int(abs(node.inputs.x.value))  # pylint: disable=invalid-name
+        self.ctx.inputs.y = Int(abs(node.inputs.y.value))  # pylint: disable=invalid-name
```

Comment:
Sure this is fine, just wondering how this came up in this PR? I don't think you've removed this file from the pre-commit ignore right?

Reply:
That's a very good question.
Notice that I introduced this in my second commit because the pre-commit hook failed.
I suspect this is because some dependency of dependency changed...